modules/_commands.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
 							}
 							if ( cmd.privileged ) {
 								if ( bot.config.irc.control !== msg.to ) {
-									bot.say( msg.to, msg.nick + ': Unable.' );
+									bot.say( msg.to, "I'm sorry, " + msg.nick + "I'm afraid I can't let you do that." );
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/douglas/civilservant/file/2351755039/issues/source?bid=3060616&fileBranchId=3217890#l25)